shrinkpdf.sh

```diff
@@ -71,6 +71,15 @@ get_pdf_version ()
 	fi
 }
 
+
+check_input_file () {
+	# Check if $ifile exists
+	if [ ! -f "$ifile" ]; then
+		echo "Error: Input file does not exist." >&2
```

Comment:
Maybe we should add the filename to the error message? Probably not necessary for a small script like this.